renderdoc/driver/shaders/spirv/spirv_reflect.cpp

```diff
@@ -300,8 +300,8 @@ static bool IsStrippableBuiltin(rdcspv::BuiltIn builtin, bool perPrimitive)
 {
   if(perPrimitive &&
      (builtin == rdcspv::BuiltIn::PrimitiveId || builtin == rdcspv::BuiltIn::Layer ||
-      builtin == rdcspv::BuiltIn::ViewportIndex || builtin == rdcspv::BuiltIn::CullPrimitiveEXT ||
-      builtin == rdcspv::BuiltIn::ShadingRateKHR))
+      builtin == rdcspv::BuiltIn::ViewIndex || builtin == rdcspv::BuiltIn::ViewportIndex ||
+      builtin == rdcspv::BuiltIn::CullPrimitiveEXT || builtin == rdcspv::BuiltIn::ShadingRateKHR))
```

Comment:
I don't think `ViewIndex` should be included here. This function is checking for the `gl_PerVertex` block, or `gl_MeshPerVertexEXT `/`gl_MeshPerPrimitiveEXT` blocks, which might be generated as a whole block even if only one component is used. The multiview index will never be part of that so shouldn't be marked as strippable.